magda-gateway/src/createBaseProxy.ts

```diff
@@ -69,17 +69,20 @@ export default function createBaseProxy(tenantMode: TenantMode): httpProxy {
         }
     });
 
-    proxy.on("proxyRes", function(proxyRes, req, res) {
+    proxy.on("proxyRes", function (proxyRes, req, res) {
         // Add a default cache time of 60 seconds on GETs so the CDN can cache in times of high load.
-        if (
-            req.method === "GET" &&
-            !proxyRes.headers["Cache-Control"] &&
-            !proxyRes.headers["cache-control"] &&
-            !req.headers["Cache-Control"] &&
-            !req.headers["cache-control"]
-        ) {
-            proxyRes.headers["Cache-Control"] = "public, max-age=60";
-        }
+
+        // FIXME FIXME FIXME TODO: Add this back and make it configurable.
```

Comment:
Better fix this first!

Reply:
I thought the disabling cache from client side was working.
Do we have a reproducible case?

Reply:
It does for the bits we put it in, but I didn't want to discover a place where we'd missed it once we'd already got the demo out. The only time I noticed this being a problem was when navigating back to the home page from Add Dataset, I don't think it knows to not cache then.

Either way I think this should be configurable though.

Reply:
(which is something I'll do :+1: )

Reply:
Fetch API seems support a `cache` option (not the cache header):
https://developer.mozilla.org/en-US/docs/Web/API/Request/cache
the `no-cache` mode seems could 100% prevent client read data from the cache.